salt/environment_settings.yml

```diff
@@ -229,7 +242,75 @@ environments:
         instances:
           edx:
             number: 1
-            type: t2.medium
+            type: t3.medium
+  mitxpro-production:
+    business_unit: mitxpro
+    network_prefix: '10.8'
+    vpc_name: MITxPro Production
+    secret_backends:
+      mysql:
+        role_prefixes:
+          - edxapp
+          - edxapp-csmh
+          - xqueue
+      mongodb:
+        role_prefixes:
+          - modulestore
+          - contentstore
+          - gitlog
+          - forum
+      rabbitmq:
+        role_prefixes:
+          - celery
+      aws:
+        bucket_prefix: xpro
+        bucket_uses:
+          - grades
+          - storage
+    backends:
+      pki:
+        - consul
+        - elasticsearch
+        - fluentd
+        - mongodb
+        - rabbitmq
+      rds:
+        - name: mitxproproduction
+          multi_az: False
+          engine: mariadb
+          vault_plugin: mysql-database-plugin
+          password_length: 40
+          db_instance_class: db.m5.large
+          allocated_storage: 250
+          public_access: False
+          engine_version: '10.3'
+          purpose: xpro-production
+          service_name: mysql
+          mount_point: mysql-mitxpro-production
+      elasticache:
+        - engine: memcached
+          engine_version: '1.4.34'
+          node_type: cache.r5.large
+          num_cache_nodes: 1
+          cluster_id: xpro-edxapp
+          purpose: xpro-production
+    purposes:
+      xpro-production:
+        app: edx-platform
+        business_unit: mitxpro
+        domains:
+          cms: studio-xpro-qa.mitx.mit.edu
+          lms: xpro-qa.mitx.mit.edu
+          preview: preview-xpro-qa.mitx.mit.edu
+        versions:
+          <<: *xpro_production_versions
+        instances:
+          edx:
+            number: 6
+            type: r5.large
```

Comment:
Why `r5` and not `t3`?

Reply:
Because we need to make sure that we have enough headroom for production workloads, similar to what we use on residential production.